odhseldon/README.md

```diff
@@ -0,0 +1,20 @@
+# Seldon
+
+Seldon component subscribes to Seldon certified operator, which in turn issues an install of the Seldon certified operator in the namespace. 
+ 
+### Folders
+There are is one main folders in the Seldon component
```

Comment:
folder instead of folders

Reply:
This will stay folders, it is a title of the section listing Folders, in this case it is only one folder

Reply:
I meant where it says "There are is one main folders".......Should be, "There is one main folder"